src/themes/teams/components/Input/inputStyles.ts

```diff
@@ -14,22 +15,35 @@ const inputStyles: ComponentSlotStylesInput<InputProps, InputVariables> = {
 
   input: ({ props: p, variables: v }): ICSSInJSStyle => ({
     outline: 0,
-    border: 0,
+    border: v.border,
     borderRadius: v.borderRadius,
     borderBottom: v.borderBottom,
     color: v.fontColor,
     backgroundColor: v.backgroundColor,
+    position: 'relative',
     padding: v.inputPadding,
     ...(p.fluid && { width: '100%' }),
     ...(p.inline && { float: 'left' }),
+    '::placeholder': {
+      color: v.fontColor,
+    },
     ':focus': {
-      borderColor: v.inputFocusBorderColor,
+      borderBottomColor: v.inputFocusBorderColor,
+    },
+    '&::after': {
       borderRadius: v.inputFocusBorderRadius,
+      borderBottom: `${pxToRem(2)} solid transparent`,
+      content: '',
```

Comment:
In CSS, the value required for `content` is quoted.  However, in JS, we need to quote the CSS values 😝 

```suggestion
      content: '""',
```

We should really consider implementing a warning for this in dev mode.  Just add a fela plugin that is wrapped in a NODE_ENV check for dead code elimination.  That plugin will log a warning if the `content` value is not quoted.

@alinais / @kuzhelov / @miroslavstastny

Reply:
fixed